src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -152,11 +179,36 @@ public final void refreshItems() {
 
     }
 
+    private int getClosedIssuesCount() {
+        return (int) getElementsList().stream().filter((element) -> !element.getIssue().isOpen()).count();
+    }
```

Comment:
@ndt93 is there a better place for this function? Generating the stats should be done closer to the data rather than the UI?